41.md

```diff
@@ -0,0 +1,114 @@
+NIP-41
+======
+
+Stateless Unambiguous Key Invalidation
+--------------------------------------
+
+`draft` `optional` `author:fiatjaf` `author:RubenSomsen`
+
+The idea of this NIP is that compromised keys can be invalidated in a way that prevents identity theft.
+
+Cryptographers must forgive me for trying to write this in a way that humans like myself can understand.
+
+## Notation
+
+Lowercase letters denote private keys, uppercase letters denote public keys. `||` means concatenate. `sk_` means "secret key", `pk_` means "public key". `G` is the generator point of the secp256k1 curve.
```

Comment:
This explanation mentions lowercase and uppercase notations, whereas later `A` and `A'` is used, which is confusing.

Reply:
Lowercase is private key, uppercase is public key. `'` is key derived from the non-`'`.

Reply:
I'll improve it.